frontend/src/scenes/insights/trendsLogic.ts

```diff
@@ -224,6 +241,25 @@ export const trendsLogic = kea<trendsLogicType<FilterType, ActionType, TrendPeop
                 setShowingPeople: ({}, { isShowing }) => isShowing,
             },
         ],
+        indexedResults: [
+            [],
+            {
+                setIndexedResults: ({}, { results }) => results,
+            },
+        ],
+        visibilityMap: [
+            {} as Record<number, any>,
+            {
+                setVisibility: (state: Record<number, any>, { entry }: { entry: Record<number, any> }) => ({
```

Comment:
Note: The name is a misnomer - I'd assume setVisibility would overwrite all, but this does a merge.